apps/user_status/src/components/SetStatusModal.vue

```diff
@@ -25,15 +26,22 @@
 
 			<!-- Status message form -->
 			<form @submit.prevent="saveStatus" @reset="clearStatus">
-				<div class="set-status-modal__header">
-					<h2>{{ $t('user_status', 'Status message') }}</h2>
-				</div>
+				<h3 class="set-status-modal__header">
+					{{ $t('user_status', 'Status message') }}
+				</h3>
 				<div class="set-status-modal__custom-input">
 					<CustomMessageInput ref="customMessageInput"
 						:icon="icon"
 						:message="editedMessage"
 						@change="setMessage"
 						@select-icon="setIcon" />
+					<NcButton v-if="messageId === 'vacationing'"
+						:href="absencePageUrl"
+						target="_blank"
+						type="secondary"
+						:aria-label="$t('user_status', 'Set absence period')">
+						{{ $t('user_status', 'Set absence period and replacement') + ' ↗️' }}
```

Comment:
Sorry I meant this that we use in other places:
https://github.com/nextcloud/external/blob/296bb81e55433e8bedca14f75f463ea860400a76/templates/settings.php#L30
```suggestion
						{{ $t('user_status', 'Set absence period and replacement') + ' ↗' }}
```